content/reference/current/user/overview/index.md

```diff
@@ -3,25 +3,25 @@ title: "Overview"
 description: "Learn how to navigate."
 lead: "Gatling Enterprise navigation"
 date: 2021-03-10T09:29:04-05:00
-lastmod: 2021-08-16T17:55:36+02:00
+lastmod: 2021-11-23T12:00:00+02:00
 weight: 2020
 ---
 
-Once you are logged in, you are now able to navigate using the Gatling Enterprise navigation bar on the left side.
+Once you are logged in, you can navigate using the Gatling Enterprise navigation bar on the left side.
 
 {{< img src="overview.png" alt="Overview" >}}
 
 Gatling Enterprise is composed of:
 
-On top right:
-- **The Profile section**, letting a user manage his account settings (not accessible to the superAdmin)
+On the top right:
+- **The Profile section**, where a user can manage their own account settings (not accessible to the superAdmin)
```

Comment:
Do the customers on the on-premises version have access to the superAdmin account?